src/main/java/org/openelisglobal/audittrail/daoimpl/AuditTrailServiceImpl.java

```diff
@@ -465,17 +527,16 @@ private String getChanges(BaseObject newObject, BaseObject existingObject, Strin
                     } else {
                         LogEvent.logInfo(this.getClass().getSimpleName(), "getChanges",
                                 "Field " + fields[ii].getName() + " values differ, processing change");
-                        LabelValuePair lvb = processLabelValue(fieldName, propertyPreUpdateState, existingObject,
-                                newObject);
```

Comment:
any reason you modified this service ?? did you test all places where this is used and it doesnt break ??

Reply:
For some reason, previously the audit trail only captured the old value when a field changed. With these changes i try to capture both old and new values, which provides more complete and sensible audit history